libraries/AP_GPS/GPS_Backend.h

```diff
@@ -95,7 +95,7 @@ class AP_GPS_Backend
 
     // return iTOW of last message, or zero if not supported
     uint32_t get_last_itow(void) const {
-        return (_pseudo_itow_delta_ms == 0)?(_last_itow*1000ULL):((_pseudo_itow/1000ULL) + _pseudo_itow_delta_ms);
+        return (_pseudo_itow_delta_ms == 0)?(_last_itow):((_pseudo_itow/1000ULL) + _pseudo_itow_delta_ms);
```

Comment:
Do you think we should rename `_last_itow` to `_last_itow_ms`?  That may have stopped this bug creeping in.